lib/core/plugin.rb

```diff
@@ -144,22 +145,29 @@ def help
   # @return string representing response to be displayed
   def exec(data_from_chat = nil)
     # based on some meta information like the type then execute the proper way
+    # Strip an incorrection coded qoutes to UTF8 ones
     data_from_chat.text.tr!('“', '"')
```

Comment:
For completeness should we either replace the other quote and/or log an issue to look into this farther so we don't lose track of it.